api/models/Tag.js

```diff
@@ -53,6 +53,12 @@ const addToCommunity = (community, tag, user_id, trx) => {
       user_id: user_id,
       created_at: new Date()
     }).save({}, {transacting: trx})
+    .then(() => new TagFollow({
```

Comment:
a big change here: now when you create a tag you are automatically following it. This means we have new_post_counts for the created tags in the left nav.
You can still unfollow on the tag page, which just means your created but not followed tag will never be highlighted in the left menu.
This seemed preferable to adding a new_post_count to community tags and duplicating a lot of stuff.